test/test_eigen.js

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai';
+import { leadingEigenvalue } from '../src/eigenvalues.js';
+import { flattenNested } from '../src/utils.js';
+import BRA from '../data/BRA.json';
+
+describe('leadingEigenvalue', function() {
+  it('works for a non-symmetric matrix', function() {
+    expect(
+      leadingEigenvalue([[1, 2], [3, 4]])
+    ).to.be.closeTo(5.3722813, 1e-6);
+  });
+
+  it('is fast', function() {
+    this.timeout(1000);
```

Comment:
I had a little fight but could not get the old version to easily run here as a comparison - how much faster is this?

Reply:
Ah, so the build before did the Brazil test in [6.4s](https://github.com/mrc-ide/squire_js/runs/1459029323?check_suite_focus=true#step:4:80)
And this build does it in [.4s](https://github.com/mrc-ide/squire_js/runs/1464364460?check_suite_focus=true#step:4:81)

So probably ~16x?